src/main/java/com/epam/config/DBConfiguration.java

```diff
@@ -28,4 +29,9 @@ public DataSource dataSource() {
     public NamedParameterJdbcTemplate namedParamJdbcTemplate() {
         return new NamedParameterJdbcTemplate(dataSource());
     }
+
+    @Bean
```

Comment:
Do we need both jdbcTemplate and namedParamJdbcTemplate beans?

Reply:
done in another pr